gendiff/scripts/gendiff.py

```diff
@@ -16,7 +17,49 @@ def main():
         help="set format of output",
     )
     args = parser.parse_args()
-    print(parser, args)
+    return args
+
+
+def generate_diff(first_file, second_file):
+    with (
+        open(first_file, "r") as file1,
+        open(second_file, "r") as file2,
+    ):
+        f1 = json.load(file1)
+        f2 = json.load(file2)
```

Comment:
it's better to get access to the resource for a short period of time, e.g. here you don't need to have files opened after you convert them into dicts  

code starting from line 30 can be moved outside of this context manager block